app/views/application/_subnavigation.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class='govuk-grid-row subnav govuk-!-margin-0'>
   <div class='govuk-grid-column-one-third govuk-grid-column-one-half-from-desktop govuk-!-padding-left-0 organisation-name'>
     <p class='govuk-body govuk-!-margin-0 govuk-!-padding-0'>
-      <strong class="govuk-!-margin-right-1"><%= current_organisation.name %></strong>
+      <strong class="govuk-!-margin-right-1"><%= current_organisation.name if current_organisation %></strong>
```

Comment:
Also no need to modify these files as we will redirect to the public page.
We can safely assume that anyone who is logged in has an organisation at this point.